graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -450,21 +523,64 @@ abstract class BaseDataTypeGenerator(
                 .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
                 .build()
 
+        if(config.generateBitset) {
```

Comment:
the method `addBuilder` looks too long at this point. I think it's worth refactoring it into smaller methods.